packages/components/src/color-picker/picker.tsx

```diff
@@ -15,12 +15,14 @@ interface PickerProps {
 }
 
 export const Picker = ( { color, enableAlpha, onChange }: PickerProps ) => {
-	const Component = enableAlpha ? HslaColorPicker : HslColorPicker;
-	const hslColor = useMemo( () => color.toHsl(), [ color ] );
+	const Component = enableAlpha
```

Comment:
While I understand why we switched from an object-based to a string-based picker, I don't understand why we switched also from an HSL-based to an RBG-based picker — could you explain why?

Reply:
The format we use outside the component is hex code which maps directly to RGB just by converting hex to decimal. Since we are changing to a string-based picker why not use the RGB one when it maps more directly? What advantage do we get in using an HSL string-based picker?

Reply:
Ok, it looks like the only difference would be about the input string format — while the underlying `ColorPicker` would still be the same. I guess using a HSL picker made sense previously, when this component used HSL internally to represent color values — but with the current state of the component, this change should be fine :)

Thank you for explaining!